modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,120 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 
+[id="journald-receiver_{context}"]
+=== Journald Receiver
+
+:FeatureName: The Journald Receiver
+include::snippets/technology-preview.adoc[]
+
+The Journald Receiver parses *journald* events from the *systemd* journal and sends them as logs.
+
+.OpenTelemetry Collector custom resource with the enabled Journald Receiver
+[source,yaml]
+----
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: otel-journald
+  labels:
+    security.openshift.io/scc.podSecurityLabelSync: "false"
+    pod-security.kubernetes.io/enforce: "privileged"
+    pod-security.kubernetes.io/audit: "privileged"
+    pod-security.kubernetes.io/warn: "privileged"
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: privileged-sa
+  namespace: otel-journald
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-journald-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:openshift:scc:privileged
+subjects:
+- kind: ServiceAccount
+  name: privileged-sa
+  namespace: otel-journald
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel-journald-logs
+  namespace: otel-journald
+spec:
+  mode: daemonset
+  serviceAccount: privileged-sa
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+      - CHOWN
+      - DAC_OVERRIDE
+      - FOWNER
+      - FSETID
+      - KILL
+      - NET_BIND_SERVICE
+      - SETGID
+      - SETPCAP
+      - SETUID
+    readOnlyRootFilesystem: true
+    seLinuxOptions:
+      type: spc_t
+    seccompProfile:
+      type: RuntimeDefault
+  config: |
+    receivers:
+      journald:
+        files: /var/log/journal/*/*
+        priority: info # <1>
+        units: # <2>
+          - kubelet
+          - crio
+          - init.scope
+          - dnsmasq
+        all: true # <3>
+        retry_on_failure:
+          enabled: true # <4>
+          initial_interval: 1s # <5>
+          max_interval: 30s # <6>
+          max_elapsed_time: 5m # <7>
+    processors:
+    exporters:
+      debug:
+        verbosity: detailed
+    service:
+      pipelines:
+        logs:
+          receivers: [journald]
+          exporters: [debug]
+  volumeMounts:
+  - name: journal-logs
+    mountPath: /var/log/journal/
+    readOnly: true
+  volumes:
+  - name: journal-logs
+    hostPath:
+      path: /var/log/journal
+  tolerations:
+  - key: node-role.kubernetes.io/master
+    operator: Exists
+    effect: NoSchedule
+EOF
+----
+<1> Filters output by message priorities or priority ranges. Defaults to `info`.
+<2> Lists the units to read entries from. If empty, entries are read from all units.
+<3> Includes very long logs and logs with unprintable characters. Defaults to `false`.
+<4> If set to `true`, the receiver pauses reading a file and attempts to resend the current batch of logs when encountering an error from downstream components. Defaults to `false`.
```

Comment:
```suggestion
<1> The priorities or priority ranges to filters output by . The defaults is `info`.
<2> The list of units to read entries from. If empty, entries are read from all units.
<3> If the specified value is `true`, very long logs and logs with unprintable characters are included. The default is `false`.
<4> If set to `true`, the receiver pauses reading a file and attempts to resend the current batch of logs when encountering an error from downstream components. The default is `false`.
```

Reply:
Thank you. I applied some of these suggestions.